tests/unit/style/rule_tree/bench.rs

```diff
@@ -148,6 +152,7 @@ fn bench_expensive_insertion(b: &mut Bencher) {
 #[bench]
 fn bench_insertion_basic_parallel(b: &mut Bencher) {
     let r = RuleTree::new();
+    thread_state::initialize(thread_state::SCRIPT);
```

Comment:
I think this would also be fine with `thread_state::LAYOUT`, and would be preferable.